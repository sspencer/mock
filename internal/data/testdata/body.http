### Create loaf
# @status=201
post /bread
Content-Type: application/json

{
  "name": "value"
}
