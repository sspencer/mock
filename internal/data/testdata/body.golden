Create loaf: POST /bread delay=0s status=201 header=28 body=23